### ISSUE TITLE
Send Client Hint headers to the endpoints via JS and JSON

### ISSUE THREAD

**Issue by kirsring** [51Degrees/cloud-issues#18]
An update to the cloud documentation (https://cloud.51degrees.com/api-docs/index.html) has been identified on how to send the User-Agent Client Hint headers to the endpoints via JS and JSON. How to do this is documented on this GH issue. We will be adding this to our documentation in due time. 

We suggest that you use the JavaScript end point which handles all of this for you. An example URL which provides bespoke JavaScript for the browser and Resource Key would be:
`https://cloud.51degrees.com/api/v4/[your resource key].js`

If you are using the service server side, then we recommend using the APIs that all support cloud configuration. These handle caching and other common issues.

If you do wish to call the JSON endpoint directly then you need to provide the evidence in the parameters of the request. Here is an example of an encoded URL for JSON:

`https://cloud.51degrees.com/api/v4/[your resource key].json?sec-ch-ua=Microsoft+Edge;v=113,+Chromium;v=113,+Not-A.Brand;v=24&sec-ch-ua-mobile=?1&sec-ch-ua-platform=Android&sec-ch-ua-platform-version=10&sec-ch-ua-model=SM-G981B&sec-ch-ua-full-version-list=Microsoft+Edge;v=113.0.1774.50,+Chromium;v=113.0.5672.127,+Not-A.Brand;v=24.0.0.0&user-agent=Mozilla/5.0+(Linux;+Android+10;+SM-G981B)+AppleWebKit/537.36+(KHTML,+like+Gecko)+Chrome/80.0.3987.162+Mobile+Safari/537.36+Edg/113.0.0.0`

Basically, the parameters of the request need to be the HTTP header name and then the value just like the user-agent example in the cloud documentation.

Alternatively, you could use the 51Degrees UAParser – just another option! https://github.com/51Degrees/ua-parser-js

If this solution does not work, please reply to this GH issue. Thanks!
